controllers/scan.js

```diff
@@ -2,16 +2,18 @@
 
 const sha1 = require("../sha1-utils");
 const HIBP = require("../hibp");
+const TIPS = require("../tips");
```

Comment:
Why not just import this as `passwordTips`? Bonus points, you could then use object shorthand syntax below (if you wanted).